processor/src/utils.rs

```diff
@@ -1,9 +1,9 @@
-use super::{Felt, Vec};
+use super::Felt;
 
 // RE-EXPORTS
 // ================================================================================================
 
-pub use vm_core::utils::*;
+pub use vm_core::utils::{collections::*, *};
```

Comment:
Would this create redundant public exports? Meaning, both `utils::collections` and `collections` would be exported. If so, I would separate these as:
```Rust
use vm_core::utils::collections::*;

pub use vm_core::utils::*;
```

Reply:
changed